cl/users/admin.py

```diff
@@ -19,30 +21,42 @@
     UserProfile,
 )
 
+UserProxyEvent = apps.get_model("users", "UserProxyEvent")
+UserProfileEvent = apps.get_model("users", "UserProfileEvent")
 
-def get_email_confirmed(obj):
-    return obj.profile.email_confirmed
-
-
-get_email_confirmed.short_description = "Email Confirmed?"
 
+class TokenInline(admin.StackedInline):
+    model = Token
 
-def get_stub_account(obj):
-    return obj.profile.stub_account
 
+class UserProfileInline(admin.StackedInline):
+    model = UserProfile
 
-get_stub_account.short_description = "Stub Account?"
 
+class CustomUserChangeForm(UserChangeForm):
+    class Meta(UserChangeForm.Meta):
+        model = User
+        fields = "__all__"
 
-class TokenInline(admin.StackedInline):
-    model = Token
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Ensure user_permissions field uses an optimized queryset
+        if "user_permissions" in self.fields:
+            self.fields["user_permissions"].queryset = (
+                Permission.objects.select_related("content_type").order_by(
```

Comment:
Is there a reason to add a custom `order_by` here? 

Inspecting the order of permissions, it seems this custom order is fragmenting the permissions for the same model.

![Screenshot 2024-12-26 at 3 22 59 p m](https://github.com/user-attachments/assets/2d8ec03d-f377-47da-8435-b51f98502ae2)

Without this custom order_by, permissions for the same model are shown together.

![Screenshot 2024-12-26 at 3 23 30 p m](https://github.com/user-attachments/assets/3c203978-0709-4ccd-aae1-51522369c7a2)